### ISSUE TITLE
Service fails to start with ModuleNotFoundError for openrgb

### ISSUE THREAD

**Issue by aurtzy** [zocker-160/keyboard-center#13]
Just updated to 0.1.18-testing. Service seems to be failing due to openrgb not being found:
```
Oct 01 18:00:26 AL-PC systemd[577]: Started background service for the Keyboard-Center application.
Oct 01 18:00:26 AL-PC python3[586]: Traceback (most recent call last):
Oct 01 18:00:26 AL-PC python3[586]:   File "/usr/lib/keyboard-center/service.py", line 23, in <module>
Oct 01 18:00:26 AL-PC python3[586]:     from lib.openrgb.orgb import OpenRGBClient
Oct 01 18:00:26 AL-PC python3[586]:   File "/usr/lib/keyboard-center/lib/openrgb/__init__.py", line 3, in <module>
Oct 01 18:00:26 AL-PC python3[586]:     from openrgb import utils
Oct 01 18:00:26 AL-PC python3[586]: ModuleNotFoundError: No module named 'openrgb'
Oct 01 18:00:26 AL-PC systemd[577]: keyboard-center.service: Main process exited, code=exited, status=1/FAILURE
Oct 01 18:00:26 AL-PC systemd[577]: keyboard-center.service: Failed with result 'exit-code'.
```  

This was checked after I had restarted the service, and rebooting the system does not fix the issue. OpenRGB is not installed.

**Comment by zocker-160** [comment]
ooh I messed the import up, sorry about that!

will fix asap

**Comment by zocker-160** [comment]
should be fixed in 0.1.19
